packages/camel-catalog/src/json-schema-to-typescript.mts

```diff
@@ -53,7 +53,16 @@ async function main() {
   const exportedFiles: string[] = [];
 
   console.log('---');
-  const targetSchemaNames = ['camelYamlDsl', 'Integration', 'Kamelet', 'KameletBinding', 'Pipe'];
+  const targetSchemaNames = [
+    'camelYamlDsl',
+    'Integration',
+    'Kamelet',
+    'KameletBinding',
+    'Pipe',
+    'ObjectMeta',
+    'ObjectReference',
+    'pipe-errorHandler',
```

Comment:
[extremely-minor][:nail_care:] Should we rename it to `PipeErrorHandler` instead?
```suggestion
    'PipeErrorHandler',
```

Reply:
done :+1: